### PR TITLE
Add a value about prioritizing the user experience

### PR DESCRIPTION
**Pull request by jansepar**

### CODE REVIEW

**Comment by ericmuyser** [comment]
Glanced over it, looks good to me!

**Comment by mikenikles** [comment]
Makes sense 👍

File: README.md [hunk 1]
```diff
@@ -72,6 +72,12 @@ our company and culture evolve, so should this document.
 - Tests also help us write better code. Nothing helps refactor a
   giant method better than writing a test for it!
 
+#### Prioritize End-User Experience over Developer Experience
+
+Sometimes, it's tempting to improve the codebase at all costs, but there is one cost that should never be
+compromised: the user experience. Ensure code changes you made to improve the developer experience don't
```
Comment (kschmidtdev) [inline README.md]: I would argue to use `make` over `made`

Comment (jansepar) [inline README.md]: Good point Karl I'll make the change even though this is merged!

**Pull request merged**